luxonis_train/nodes/heads/efficient_keypoint_bbox_head.py

```diff
@@ -214,3 +215,16 @@ def _process_to_bbox_and_kps(
             max_det=self.max_det,
             predicts_objectness=False,
         )
+
+    def get_custom_head_config(self) -> dict:
+        """Returns custom head configuration.
+
+        @rtype: dict
+        @return: Custom head configuration.
+        """
+        return {
+            "iou_threshold": self.iou_thres,
+            "conf_threshold": self.conf_thres,
+            "max_det": self.max_det,
+            "n_keypoints": self.n_keypoints,
+        }
```

Comment:
Let's also add:
`"subtype": "yolov6"`

Generally these parameters should match the ones specifid in the respective parser in [depthai-nodes](https://github.com/luxonis/depthai-nodes/tree/main/depthai_nodes/ml/parsers)